### DISCUSSION TITLE
New Biome logo

### DISCUSSION THREAD

**Discussion by ematipico** [biomejs/biome#141] [Ideas]
Hi, Biomers!

I want to open this discussion to talk about a possible new logo for Biome!

I have been reading about some discontent with the current logo and agree with a few of them. Some people already started to propose some alternatives, and I would like to open a discussion in case anyone would like to propose something different or better.

Here's some pointers:
- we have a [repository](https://github.com/biomejs/resources) that contains all our assets. That should give you an idea of the number/type of assets we would need;
- the slogan "Toolchain of the web" should stay in some version of the new logo;
- I like orange/yellow, but it's a personal preference, and feel free to be creative;
- We originally liked the letter as a logo, but I'm no designer, so I am sure there are better patterns out there that I am not aware of;

Please submit your proposal by **creating a new comment** so we can start the discussion by replying to that comment.

Once (if) we choose the new logo, the creator will have their social contacts highlighted in the resources repository and the credits section of the website with its own heading.

**Comment by arb85** [comment]
I don't know if I’ve missed it, but there is a document or a text explaining why you chose the name Biome?

Maybe it can help in designing the logo and the icon..

**Reply by stephwestgarth** [reply]
I hope to be corrected if I'm wrong, the blog post essentially said because it's a play on "bis" (meaning 2/again) and "rome", aka Biome is Rome 2. https://biomejs.dev/blog/annoucing-biome

**Reply by arb85** [reply]
Thank you @stephwestgarth !!

**Reply by natemoo-re** [reply]
I love that "Biome" is also the word for a self-contained ecosystem, which exactly is what the project aims to be!

**Comment by stephwestgarth** [comment]
I have no art skills but I wonder if there's something we could do around pruning shears etc, seeing as we're linting and formatting code like someone would prune and tidy a garden to help it thrive

**Reply by ematipico** [reply]
You might be onto something. Sometimes, I refer to Biome as a tool to maintain your projects "healthy".

**Comment by Conaclos** [comment]
Some basic guidelines I am thinking about:

- The logo should be compatible with dark and light themes (or provide a version for each one)
- Sources should be in SVG and, if possible, human-readable and modifiable

**Comment by natemoo-re** [comment]
<img width="1024" alt="biome" src="https://github.com/biomejs/biome/assets/7118177/137424f2-6c8a-49f9-8e87-ff5d2a727cfa">

I made this one as an exploration, but I got some feedback that it looked too much like a crypto project (which, tbh, is totally fair.) 

I was hoping to evoke the idea of the JS logo inside of a "Biome" or ecosystem, but this approach was probably too minimalistic. 

IMO the existing black + yellow (or red) branding would be good to keep for continuity's sake.

**Reply by arb85** [reply]
Really like the "ecosystem" idea and the orange/yellow gradient

**Reply by o-t-w** [reply]
Which typeface is that using? Looks really nice, but yes, reminds me of a bank or crypto for some reason.

**Reply by natemoo-re** [reply]
Typeface is [MD Primer](https://mass-driver.com/typefaces/md-primer) by Mass-Driver. Big fan!

**Comment by arb85** [comment]
My two cents..
(just kidding, only some reflections on the meaning)

I'm not convinced with the result, but maybe this experiment can help to define the field..

> Meaning
![Artboard 1](https://github.com/biomejs/biome/assets/13294791/d63f22df-2f6e-44a4-b85a-824348fa68ee)

> Case
![Artboard 1 copy](https://github.com/biomejs/biome/assets/13294791/ef86427f-e655-4cc9-b2c9-7805bd9a3ab4)

> Components
![Artboard 1 copy 2](https://github.com/biomejs/biome/assets/13294791/3a5e9c46-47e7-4059-b2ec-986a5d2a7151)

> "B" experiment
![Artboard 1 copy 3](https://github.com/biomejs/biome/assets/13294791/7c231bc2-33f3-4bb9-85af-8ff0b116f588)

> Payoff
![Artboard 1 copy 4](https://github.com/biomejs/biome/assets/13294791/d1a6a1b0-5276-498e-9ea4-2b52f5adf5b3)

> Light / Dark
![Artboard 1 copy 5](https://github.com/biomejs/biome/assets/13294791/54f20b01-3c50-46e6-a6ce-574041bf4f10)

**Reply by ematipico** [reply]
Thank you for the submission!

Not a big fan of this one, it would be great if we could move over the Rome history, even though initially Biome comes from it.

**Comment by jamiebuilds** [comment]
First, get rid of the yellow. Yellow sucks to work into your brand because its hard to create enough contrast and it tends to fall into the background. Yellow is also too much of the same as Rome. And you should definitely try to be something new instead of forever holding onto Rome as a history. The only other reason for yellow is the JS logo and Biome is not just a JS tool so why would you link it that strongly.

Second, Biome is a rich word, and its link to what the tool is is obvious, I would embrace its more natural meaning to give the 'brand' some life (pun intended). I would look for inspiration in like space domes for sort of self-contained _biomes_. Imagine a little bubble dome with water and a tropical island, or a lush rainforest, or a tiny icy wonderland. What could that look like as a logo? Maybe could even have different variations on it based on the sub-tool.

**Reply by Conaclos** [reply]
Some tries:

![biome logo attempt](https://raw.githubusercontent.com/Conaclos/biome-logo/main/biome1.png)
![biome logo attempt](https://raw.githubusercontent.com/Conaclos/biome-logo/main/biome2.png)

**Reply by cdvillard** [reply]
I think this could have some legs. I see these as stylistic versions of a more generic version that highlights the reflection around the orb, some kind of base, and some kind of island feature.

**Comment by Conaclos** [comment]
I tried to create a custom logo inspired by the lighting thunder emoji. This could be improved (in particular the interior shape).

![biome icon lighning thunder](https://raw.githubusercontent.com/Conaclos/biome-logo/main/lighning-thunder-biome-icon.svg)

**Reply by ematipico** [reply]
It reminds me of the The Flash hero

**Reply by j4k0xb** [reply]
![image](https://github.com/biomejs/biome/assets/55899582/cd07b22a-38fe-429f-9252-bad9f186012d)
looks almost the same as the babel logo

**Comment by pijusz** [comment]
I'm always a big fan of using animals for any kind of new logo. Since initially projects don't stand on their own from beginning, unlike old brands/institutions.
![image](https://github.com/biomejs/biome/assets/20190386/7d54fb1a-3b31-43ad-934b-9718070fbaff)
![image](https://github.com/biomejs/biome/assets/20190386/25296428-845f-4423-a7af-1dcddf4313e0)
Silly Capitoline Wolf it is.

**_Sidenote_**: this was sketched in under an hour, if there is any interest in the design idea itself. We can start improving it. This is pretty much 'Spaghetti Design'. If Monochrome will look good, there won't be any issues adding colors. Also Vector itself needs lots of improvement, also maybe we need a serious wolf? Although love direction that Deno has.

**Comment by natemoo-re** [comment]
Another attempt 
<img width="1024" alt="biome" src="https://github.com/biomejs/biome/assets/7118177/541677ce-4a9a-41ce-b8a5-ccf0825877b5">

**Comment by lilnasy** [comment]
Quick exploration branch, I'm gonna try to make vector logos based on this vibe later.

![C193384F-4A37-4557-9329-94B307F4481C](https://github.com/biomejs/biome/assets/69170106/4f5ce048-56d9-4987-b678-0236dbef45ec)
![63961B3A-AF20-4848-9583-802E0F738C72](https://github.com/biomejs/biome/assets/69170106/d80c2916-4dfa-4213-9c61-96171c5a60a0)

_pretend it isnt a heavily used chopping board in my backyard_

**Comment by doodlemarks** [comment]
Hopefully not too late for this!

I incorporated the suggested colors but used them more sparingly only within the icon itself. You can see how this works both on dark and light backgrounds. The type is strong enough to use isolated as well but I would suggest keeping them together for most applications. 

![biome-brand](https://github.com/biomejs/biome/assets/2244813/c009eaa3-3526-45b5-8d22-a0cd07bdb569)

**Reply by agarleanu** [reply]
This looks really appealing, but it might not convey the idea of a "biome" that well. I like the sparkle, it adds to the "tech magic" feeling while also being geometrically fitting, but the "island in a globe" idea might be worth exploring more literally. 🤔

**Reply by doodlemarks** [reply]
For branding the metaphor doesn't necessarily need to hit you over the head to emphasize an idea.

But I would say the shape of the icon does clearly represent a space within an orb which to me is right on for a biome.

**Reply by lilnasy** [reply]
kinda reminds me of starlight

**Reply by doodlemarks** [reply]
@lilnasy surprised this came to mind! must be the inner shape, or you have been staring at Starlight too long ;)

**Reply by lilnasy** [reply]
Both! Although I couldn't put my finger on the inner shape until I put them side by side.

**Comment by lilnasy** [comment]
Going off of HTML, CSS, and JS logo colors.

![biome dark](https://github.com/biomejs/biome/assets/69170106/4ac318df-8552-431f-9f17-9c925baf8b3b)
![biome light](https://github.com/biomejs/biome/assets/69170106/e1048995-a16c-4893-af3d-2312a19e9d59)

**Reply by travislwatson** [reply]
The combo here between the leaves/petals and HTML/CSS/JS is fantastic. I'd love to see this same idea but with solid petals/leaves, using negative space for the boundary between leaves. I think solid leaves would scale down much better. The pathing on this one is very thin.

I am not a designer, but I did navigate to photopea.com last night:

![266661169-e1048995-a16c-4893-af3d-2312a19e9d59](https://github.com/biomejs/biome/assets/835300/24510f65-2915-41c6-a5ea-39903d36c3f3)

**Reply by lilnasy** [reply]
Good idea!

Here's rendered.

![biome black](https://github.com/biomejs/biome/assets/69170106/06506cce-37c2-4f2a-890c-2800a88160b5)
![biome light](https://github.com/biomejs/biome/assets/69170106/f0cbc542-0a5f-469f-bd2e-557923568fac)

**Comment by solisjoaquin** [comment]
This is a summary of my proposal
![2023-09-08-21-12-www figma com](https://github.com/biomejs/biome/assets/65245306/733720a3-75aa-419b-baf1-f0019a3ae9ba)

If you didn't notice :) I used circles and those colors to represent life and the ecosystem, but it is also the shape of a gladiator's head for Rome's past.

This is my proposal using the current color palette of Biome

![2023-09-08-21-13-www figma com](https://github.com/biomejs/biome/assets/65245306/54bae2d5-ea22-4806-b560-134da05a298c)

These are discarded ideas but useful for finding the final result:

![2023-09-08-21-16-www figma com](https://github.com/biomejs/biome/assets/65245306/291fafcc-4ac6-4dcb-b14b-f8812dca851b)

**Reply by solisjoaquin** [reply]
This is a version with the slogan. 
![2023-09-08-22-05-www figma com](https://github.com/biomejs/biome/assets/65245306/3f6a0682-7fcc-42e9-85c8-8d4fe234aaba)

(All my drafts are made with Figma and are not pixel-perfect)

**Reply by agarleanu** [reply]
When scaling this down the orange dot looks awkward, since it's too small and doesn't necessarily add to the design, otherwise the direction seems interesting!

**Comment by cdvillard** [comment]
I think there's a question of whether this should continue to associate with Rome. Not to dismiss its history, but to solidify a new, separate identity for Biome as a whole.

**Comment by nhedger** [comment]
I'll give it a try.

![Biome logo](https://github.com/biomejs/biome/assets/649677/16cc8565-28b2-4c49-9ea5-a6805b612be8)

**Comment by amorriscode** [comment]
Well, I'm definitely not a designer but this was too much fun to pass up. It's not pixel perfect but good enough to get the idea across. 😅 

<img width="706" alt="image" src="https://github.com/biomejs/biome/assets/16005567/01cc0d5c-8d3e-401a-9b43-571ea836957a">

**Comment by angelnext** [comment]
Wanted to give it a little try, FAR from perfect, but it was fun to make.

![biome](https://github.com/biomejs/biome/assets/79442303/aba3abe5-3d57-4a56-be5e-80b2cb5b8ef8)

**Comment by agarleanu** [comment]
I'm just adding what I sent on discord.

This is not what I would ship necessarily - it's just the main idea (there's still some spacings to get right).

I think I simplified it nicely, if this direction gains some traction I'll add the .svg files so we can tweak it together.

![1000017947.png](https://github.com/biomejs/biome/assets/33418415/a7af2832-0736-40c9-9fc8-6d202a855d3d)

**Reply by ematipico** [reply]
I like this proposal. I wonder, could you make one with the slogan, please?

**Reply by agarleanu** [reply]
Sure thing. If we were to go with this, I'd make a complete media kit (profile picture, banners, logo variants, with/without slogan, etc.)

When I get time this evening I'll make one with the slogan as well.

**Reply by doodlemarks** [reply]
I get iceberg or environmental cleanup company vibes from this.

**Comment by doodlemarks** [comment]
Modified this mark a bit, chose a softer typeface for the logo type, and updated the palette:

<img width="960" alt="biome-brand" src="https://github.com/biomejs/biome/assets/2244813/37c14a6b-4abd-430a-9754-4d310c766812">

**Reply by agarleanu** [reply]
Nice! After having a try myself, I realised that the "biome" concept is so incredibly broad that it's hard to capture it in any meaningful way without adding too much detail.

Was there any particular reason for abandoning the sparkle? I think it would've looked nice too, if filled.

**Reply by lilnasy** [reply]
Love it!

**Reply by denbezrukov** [reply]
Firstly, I must say that I really like the idea behind the new logo! I was wondering if we could perhaps consider trying a version without the gradient color? Also, incorporating the yellow from our original logo might be a great way to maintain a connection to our brand's roots. Just a thought! Thank you for the wonderful design effort.

**Comment by cdvillard** [comment]
I'm not much of designer or illustrator, but I had an idea that I wanted to throw out there. I know you're looking for full proposals, but I hope you'll accept this as a free idea that an illustrator can maybe run with. Also, I work best in pencil and paper, so here we go.

![Rough sketch or proposed logo blank](https://github.com/biomejs/biome/assets/6250214/3bab2d64-924d-4ea4-8fba-66ff2bcca784)

The TL;DR on this is borrowing from the idea of tiles from real-time strategy games like Civilization or Settlers of Catan, where you work to develop cities or resources in turn-based, tile-focused gameplay. A blank version of this is presented below, containing a bare "tile" save for a simple building or structure, altogether representing a singular "biome." This could act as a more corporate version of the logo for more serious matters. 

That said, this design is flexible enough that be elevated for marketing purposes to represent all kinds of different "biomes," communicating the various ways we build out development environments. For example, if Biome meetups or conferences become a thing, the base logo can be modified similar to what Smashing Magazine does with their mascot for various events.
![An example of Smashing Magazine dressing up its mascot for various events in Antwerp, Freiburg, and elsewhere.](https://github.com/biomejs/biome/assets/6250214/fbb14247-3df1-4704-aae8-4c8f4b3752c3)

The logo can be as subtle or as zany as the designer can imagine, as long as it's within the tile. Even then, it can be further extended, same as capital-B Biome.



**Why not a sphere, or terrarium, or atom, or nature theme?**

You might not have asked, but I think we need to change it up a little, and this is a start. React's logo is very much atom-inspired, as was Github's former IDE project. Terrariums were an idea I'm actually fond of, but I recall Basecamp's former logo had a very similar feel, and it makes it harder to come up with a simple corporate mark to use when Biome must represent itself as an organization. And I don't feel that nature in itself is representative of a "biome." Technically, a city can considered a self-sustained ecosystem, and if we're being honest, our offices as well.

Ultimately, this is just an idea for anyone who wants to pick it up and run with it. Like I said, I'm not a designer.

**Reply by agarleanu** [reply]
I have attempted some stacked-type designs, but to no avail. I came up with the idea of two-level environment, basically symbolizing variety and, well... two biomes. If I make it obvious enough that they're biomes, they're too detailed. If I make them recognizable and simple, you can't really tell what you're looking at anymore.

Magazine mascots work well with this type of thing. I think this is just an element of tech-stuff design - aiming for a simple stamp - in your face, simple kind of thing.

Maybe someone can do something with this tiled idea, but my attempts hasn't yielded anything cool.

**Reply by cdvillard** [reply]
I would encourage you to try again and not treat the sketches above as a template, but more of a mode. The base is supposed to be a surface upon which a biome sits, instead of multiple biomes in a single illustration. So for example, you can think of it as a cross-section of soil or flooring, upon which you'd put a house or apartment.

The base can also be expanded some. So instead of a rough 36 square units, it can be 42 or 81 if that'll give it more space. But ultimately, the abstract version of that would be the logo; everything else is literally creative fodder.

**Comment by agarleanu** [comment]
To add to the discussion, rather than just submit my latest additions, I think we need to focus on the following things.

Making a logo that:

1. Looks good big & **small**
2. Fits the current trends
3. Is a good icon
4. Has a solid (and free for commercial use) logotype

We will probably not be the next revolutionary logo designers for this, but we don't need to be. As long as we make something that respects the fundamentals and is commonly agreed-upon as pretty, I'd say we're good.

### With that in mind, I made a _helper sheet_ that you can find here:

[DOWNLOAD HELPER SHEET (SVG)](https://drive.google.com/file/d/148bF6qf6DLCCJ9zEK1s1qEuOdEUhx4R9/view?usp=sharing)


<details>
  <summary>Here's a **preview** of what the sheet looks like:</summary>
  <img src="https://github.com/biomejs/biome/assets/33418415/bc89eefa-5f6c-424a-bf03-af4afe267be9"/>
</details>

---

With that out of the way, here's my updated submission:

![image](https://github.com/biomejs/biome/assets/33418415/b54682e3-d83f-4a5a-9339-741d59175d64)
![image](https://github.com/biomejs/biome/assets/33418415/0324949e-2ab1-443b-9bd9-6ee2c792f081)
![image](https://github.com/biomejs/biome/assets/33418415/b188d78d-1bac-428c-9e09-5a0ac6ea716b)
![image](https://github.com/biomejs/biome/assets/33418415/ebe118c4-5ec2-4169-9f2f-2b857a63c3ac)

**Reply by Conaclos** [reply]
Could you try a pink touch for the island?

**Reply by ematipico** [reply]
Lovely! I like it 🤩

**Comment by doodlemarks** [comment]
One last idea I just had for a different mark:

<img width="960" alt="biome-brand" src="https://github.com/biomejs/biome/assets/2244813/f649a3cf-99b8-4d33-8f89-132964da8221">

Similarish more abstract take as posts before for a "biome". Included examples of this scaled at different resolutions to show how it works even at a small size.

**Reply by solisjoaquin** [reply]
It looks like the Instagram logo 🤔

**Comment by Conaclos** [comment]
Based on the ideas of @natemoo-re and @ugudango, I made some variations:

![logo proposals](https://raw.githubusercontent.com/Conaclos/biome-logo/main/biome3.png)

**Comment by ematipico** [comment]
We're happy to announce that the team @biomejs/core-contributors has decided, and chose @ugudango 's [logo](https://github.com/biomejs/biome/discussions/141#discussioncomment-6994777)! 

We will contact @ugudango privately to get them involved in the project, and we will slowly start working towards rebranding the website/organisation. 

Congrats @ugudango and thank you to everyone who spent time proposing new logos! ❤️

**Reply by ematipico** [reply]
@ugudango feel free to reach out to me on Discord or Twitter :)

**Comment by Conaclos** [comment]
We now have a new logo ❤️ Thanks to everyone to provide feedbacks and proposals :)
